packages/iov-bns/src/decode.spec.ts

```diff
@@ -310,6 +321,12 @@ describe("Decode", () => {
         votingEndTime: 42424243,
         submissionTime: 3003,
         author: "tiov1qqgjyv6y24n80zyeqqgjyv6y24n80zyed9d6mt" as Address,
+        state: {
```

Comment:
Why use a different key?

Reply:
Different than `.voteState`? Because I was overwhelmed by all those terms like election, vote, voting, tally, result that seem to be mixed without any real concept. How can a "state" have a type called "result", which is not a result until the voting period is over? Why it it `votingEndTime` but `voteState`? Then the noun "vote" has like [8 distict meanings](https://www.dict.cc/?s=vote), so term adds basically nothing in this context.

I'm open for better ideas but don't want to blindly copy what is coming from weave.

Reply:
Fair enough. It would be nice to have an explicit domain model actually.